source/draw/Color.ooc

```diff
@@ -73,6 +77,8 @@ ColorYuva: cover {
 	toMonochrome: func -> ColorMonochrome { ColorMonochrome new(this y) }
 	toUv: func -> ColorUv { this toYuv() toUv() }
 	toYuv: func -> ColorYuv { ColorYuv new(this y, this u, this v) }
+	toRgb: func -> ColorBgr { this toYuv() toRgb() }
```

Comment:
Upstream runs fine. The problem is here. Return type is `Bgr` but you return `Rgb`.